.github/workflows/create-quantic-package.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         working-directory: ./packages/quantic
         run: |
           echo "$SFDX_AUTH_JWT_KEY" > server.key
-          sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_JWT_CLIENT_ID --jwtkeyfile ./server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
+          sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
```

Comment:
This one references the env variable you declared on line 7 so they need to have the same name